CorsixTH/Lua/game_ui.lua

```diff
@@ -1033,102 +1033,202 @@ local function make_tutorial_phases()
+    {
+      text = _A.tutorial.click_and_drag_to_build,               -- 4
+      speech_id = AdviserStringIds.tutorial.click_and_drag_to_build,
+    },
+    {
+      text = _A.tutorial.room_in_invalid_position,              -- 5
+      speech_id = AdviserStringIds.tutorial.room_in_invalid_position,
+    },
+    {
+      text = _A.tutorial.room_too_small,                        -- 6
+      speech_id = AdviserStringIds.tutorial.room_too_small,
+    },
+    {
+      text = _A.tutorial.room_too_small_and_invalid,            -- 7
+      speech_id = AdviserStringIds.tutorial.room_too_small_and_invalid,
+    },
+    {
+      text = _A.tutorial.room_big_enough,              -- 8
+      speech_id = AdviserStringIds.tutorial.room_big_enough,
       begin_callback = function() TheApp.ui:getWindow(UIEditRoom):startButtonBlinking(4) end,
-      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end, },
+      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end,
+    },
 
     -- 3.3) door and windows
-    _A.tutorial.place_door,                            -- 9
-    _A.tutorial.door_in_invalid_position,              -- 10
-    { text = _A.tutorial.place_windows,                -- 11
+    {
+      text = _A.tutorial.place_door,                            -- 9
+      speech_id = AdviserStringIds.tutorial.place_door,
+    },
+    {
+      text = _A.tutorial.door_in_invalid_position,              -- 10
+      speech_id = AdviserStringIds.tutorial.door_in_invalid_position,
+    },
+    {
+      text = _A.tutorial.place_windows,                -- 11
+      speech_id = AdviserStringIds.tutorial.place_windows,
       begin_callback = function() TheApp.ui:getWindow(UIEditRoom):startButtonBlinking(4) end,
-      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end, },
-    { text = _A.tutorial.window_in_invalid_position,   -- 12
+      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end,
+    },
+    {
+      text = _A.tutorial.window_in_invalid_position,   -- 12
+      speech_id = AdviserStringIds.tutorial.window_in_invalid_position,
       begin_callback = function() TheApp.ui:getWindow(UIEditRoom):startButtonBlinking(4) end,
-      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end, },
+      end_callback = function() TheApp.ui:getWindow(UIEditRoom):stopButtonBlinking() end,
+    },
 
     -- 3.4) objects
-    _A.tutorial.place_objects,                         -- 13
-    _A.tutorial.object_in_invalid_position,            -- 14
-    { text = _A.tutorial.confirm_room,                 -- 15
+    {
+      text = _A.tutorial.place_objects,                         -- 13
+      speech_id = AdviserStringIds.tutorial.place_objects,
+    },
+    {
+      text = _A.tutorial.object_in_invalid_position,            -- 14
+      speech_id = AdviserStringIds.tutorial.object_in_invalid_position,
+    },
```

Comment:
If you look at the original file, the `-- XY` numbers are nicely aligned vertically, to simplify reading.

In your edit, the numbers are not vertically aligned any more.

In general, try to avoid touching things around your modifications. Try to reduce the impact of your code changes for non-related parts as much as possible.